services/ui-src/src/types/index.ts

```diff
@@ -152,9 +152,13 @@ export interface ReportShape extends ReportKeys {
   submittedBy?: string;
   submitterEmail?: string;
   submittedOnDate?: number;
+  formTemplate?: ReportJson;
+  fieldData?: AnyObject;
```

Comment:
Request: I don't think these should be necessary anymore because this type should never have those two properties on it, right?

```suggestion
```